stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/plugin/configuration/Config.kt

```diff
@@ -32,5 +32,5 @@ public data class Config @JvmOverloads constructor(
     public val userPresence: Boolean = true,
     public val persistenceEnabled: Boolean = true,
     public val uploadAttachmentsNetworkType: UploadAttachmentsNetworkType = UploadAttachmentsNetworkType.NOT_ROAMING,
-    public val useSequentialEventHandler: Boolean = false,
+    public val useSequentialEventHandler: Boolean = true,
```

Comment:
Why this change was needed?

Reply:
Yeah, it is not a part of this PR, I just noticed this change is missing in the following PRs:
- https://github.com/GetStream/stream-chat-android/pull/4083
- #4089